src/drivers/cholqrcp.cc

```diff
@@ -0,0 +1,249 @@
+#include <cstdint>
+#include <vector>
+
+#include <RandBLAS.hh>
+#include <lapack.hh>
+#include <RandLAPACK.hh>
+
+#include <chrono>
+
+using namespace RandLAPACK::comps::util;
+using namespace blas;
+using namespace lapack;
+
+using namespace std::chrono;
+
+namespace RandLAPACK::drivers::cholqrcp {
+
+// This vesrion of the code overwrites matrix A with Q
+// Note that Q is now useless here
```

Comment:
That's the comment i did not delete in time. We used to have Q as one of the args of the function. We then decided that we shall just overwrite A with Q, which made Q argument useless, but getting rid of it would have taken some work, so i kept it around for a bit and kept the comment.